<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MissModifyInfo</class>
 <widget class="QDialog" name="MissModifyInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>产品信息修改</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="2" column="2" colspan="2">
    <widget class="QPlainTextEdit" name="pedtDesc">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="lblDesc">
     <property name="text">
      <string>产品描述:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="edtError"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblRight">
     <property name="text">
      <string>正确的</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QComboBox" name="cBoxName">
     <item>
      <property name="text">
       <string>产品型号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>序列号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>产品描述</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>备注</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblError">
     <property name="text">
      <string>错误的</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cBoxName</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>MissModifyInfo</receiver>
   <slot>s_changeName(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MissModifyInfo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>28</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MissModifyInfo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtError</sender>
   <signal>editingFinished()</signal>
   <receiver>MissModifyInfo</receiver>
   <slot>s_checkNameE()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtName</sender>
   <signal>editingFinished()</signal>
   <receiver>MissModifyInfo</receiver>
   <slot>s_checkNameR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_changeName(int)</slot>
  <slot>s_checkNameE()</slot>
  <slot>s_checkNameR()</slot>
 </slots>
</ui>
